src/Adapter/WcAddressAdapter.php

```diff
@@ -88,7 +88,7 @@ private function getAddressFields($class, string $addressType): array
         // If there's an address JSON object, use it
         if ($class instanceof WC_Order) {
             $myParcelAddress = $this->getOrderMeta($class, Pdk::get('checkoutAddressHiddenInputName'), $addressType);
-            if ($myParcelAddress) {
+            if (!empty($myParcelAddress)) {
```

Comment:
Dit is toch exact dezelfde conditie? Of is dit gedaan om het explicieter te maken? Of mis ik iets?